f/chunks.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function chunks(arr, size) {
+  size = size >> 0;
+  if ( size <= 0 || size > arr.length ) return arr;
+  const buffer = new Array(Math.ceil(arr.length / size));
```

Comment:
I see why you did this, but it’s pretty unprecedented in underdash. I have favored simplicity over efficiency so people can learn how to implement the functionality, allowing them to optimize if needed (because most of the time, the optimization is not needed).

What do you think about this:

```js
function chunks(arr, size) {
  let output = [];
  for (let i = 0; i < arr.length; i += size) {
    output.push(arr.slice(i, i + size));
  }
  return output;
}
```